packages/app/src/cli/services/web-env.ts

```diff
@@ -0,0 +1,101 @@
+import {selectApp} from './dev/select-app.js'
+import {AppInterface} from '../models/app/app.js'
+import {output, file} from '@shopify/cli-kit'
+
+export type Format = 'json' | 'text'
+interface WebEnvOptions {
+  update: boolean
+  envFile: string
+}
+
+export async function webEnv(app: AppInterface, {update, envFile}: WebEnvOptions): Promise<output.Message> {
+  if (update) {
+    return updateEnvFile(app, {envFile})
+  } else {
+    return outputEnv(app)
+  }
+}
+
+export async function updateEnvFile(
+  app: AppInterface,
+  {envFile}: Pick<WebEnvOptions, 'envFile'>,
+): Promise<output.Message> {
+  let envFileContent = null
+
+  if (await file.exists(envFile)) {
+    envFileContent = await file.read(envFile)
+
+    output.info(`Current ${envFile} is:
+${envFileContent}
+`)
+  } else {
+    output.info(`No environment file found. Creating ${envFile}`)
+  }
+
+  const selectedApp = await selectApp()
+
+  const updatedValues = {
+    SHOPIFY_API_KEY: selectedApp.apiKey,
+    SHOPIFY_API_SECRET: selectedApp.apiSecretKeys[0].secret,
+    SCOPES: app.configuration.scopes,
+  }
+
+  const newEnvFileContent = patchEnvFile(envFileContent, updatedValues)
+  await file.write(envFile, newEnvFileContent)
+
+  return output.content`Updated ${envFile} to be:
+${newEnvFileContent}
+`
+}
+
+export async function outputEnv(app: AppInterface): Promise<output.Message> {
+  const selectedApp = await selectApp()
+
+  return output.content`
+  ${output.token.green('SHOPIFY_API_KEY')}=${selectedApp.apiKey}
+  ${output.token.green('SHOPIFY_API_SECRET')}=${selectedApp.apiSecretKeys[0].secret}
+  ${output.token.green('SCOPES')}=${app.configuration.scopes}
+`
+}
+
+function patchEnvFile(envFileContent: string | null, updatedValues: {[key: string]: string}): string {
```

Comment:
That makes sense. I was a bit hesitant at first because in that module we use an external library to parse the env file and in this function we use a custom approach. But the two use cases should be clear from the naming.